doc/contents.rst

```diff
@@ -36,6 +36,8 @@ Sphinx documentation contents
    authors
    code_of_conduct
 
+   reference/index
```

Comment:
Is plural form (`references/index`) better?

Reply:
It's a "reference guide", so reference is correct

Reply:
understand.